examples/IRMQTTServer/IRMQTTServer.ino

```diff
@@ -1121,6 +1125,10 @@ uint32_t maxSketchSpace(void) {
 #endif  // defined(ESP8266)
 }
 
+#if REPORT_VCC
+String vccToString(void) { return String(ESP.getVcc() / 1000.0); }
```

Comment:
I'm just eye-balling, but the uint16 will be cast to a float before the division? (ie: dividing by a float will do the cast of the int first?) I can't remember, and just trying to make sure it's all as expected :)

Reply:
It does the right thing when I tested it on a device. E.g. 3.007 etc